capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+
+interface CollegeScoreCardInterface {
+  /*General Use Functions*/
+  public boolean validateCollegeName(String collegeName);
+  public List<UniversityData> getCollegesData(List<String> collegeNames);
+
+  /*Search Feature Functions*/
+  public List<UniversityData> getCollegeSuggestions(String currName);
+
+  /*Maps Feature Functions*/
+  public List<UniversityData> updateMapData(List<UniversityData> collegeDatum);
```

Comment:
If this is just returning the data that will be used to create the map then maybe call it `getMapData`.

Reply:
thanks! done!